Update documentation for Digital Ocean - HA PR #89800 (#30166)
home-assistant/home-assistant.github.io: source/_integrations/digital_ocean.markdown

  required: true
  type: list
{% endconfiguration %}

## DNS Service

> This integration offers a service to execute {% term DNS %} record updates on any domain hosted on your [DigitalOcean Networking](https://cloud.digitalocean.com/networking/domains) section.

Reviewer: Looks like you've used the glossary/term template for a term that's not yet in the glossary. It would need to be added to `/source/_data/glossary.yml`

Author: I've added those now.

No additional configuration is required.

> Currently only A and CNAME records are supported. The service can be called either from the Developer-Tools UI, as well as from within integrations.

Reviewer: ```suggestion
Currently only A and CNAME records are supported. The service can be called either from the Developer Tools UI, or from within integrations.
```
Slight suggestion

### Service Details

| Service data attribute | Optional | Description | Example |
| ---------------------- | -------- | ----------- | ------- |
| `domain` | no | String indicating what domain you wish to update. | example.com |
| `record` | no | String what record, within this domain, to update. | home (for updating home.example.com) |
| `value` | no | String indicating the desired record value. This needs to be an {% term IPV4%} address or another {% term domain %}. | 145.12.122.41 |
| `type` | yes | The type of record to update. A and CNAME are supported. Defaults to A | A |

### Examples

> Example call within the Developer-Tools UI:

Reviewer: ```suggestion
Example call within the Developer Tools UI:
```

Author: Done.
